advisories/github-reviewed/2024/08/GHSA-mg8j-w93w-xjgc/GHSA-mg8j-w93w-xjgc.json

```diff
@@ -32,7 +32,26 @@
               "introduced": "8.0.0"
             },
             {
-              "last_affected": "11.0.4"
+              "fixed": "10.3.6"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "Packagist",
+        "name": "drupal/drupal"
```

Comment:
Should this be "drupal/core" instead of "drupal/drupal"?

Reply:
I don't think this can harm anything and better to be over-specified than under-specified, so going to leave this as-is.